src/HealthChecks.Oracle/OracleHealthCheckOptions.cs

```diff
@@ -14,6 +14,11 @@ public class OracleHealthCheckOptions
     /// </summary>
     public string ConnectionString { get; set; } = null!;
 
+    /// <summary>
+    /// Optional credential to use when connecting to the database.
```

Comment:
```suggestion
    /// Optional credential to use when connecting to the Oracle database.
```

to trigger ci, for some reason ci was not run

Reply:
Maybe because license/cla bot interactions.